gensim/test/test_fasttext.py

```diff
@@ -818,12 +818,12 @@ def test_cbow_hs_against_wrapper(self):
 
         tmpf = get_tmpfile('gensim_fasttext.tst')
         model_wrapper = FT_wrapper.train(ft_path=self.ft_path, corpus_file=datapath('lee_background.cor'),
-                                         output_file=tmpf, model='cbow', size=50, alpha=0.05, window=5, min_count=5,
+                                         output_file=tmpf, model='cbow', size=48, alpha=0.05, window=5, min_count=5,
```

Comment:
Hanging indent please (here and everywhere).

Not sure how this sneaked in, I thought we already fixed all of them with @mpenkov recently.